client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html

```diff
@@ -36,7 +36,7 @@
         </div>
         <div class="hterm">
             <div data-ng-if="OpenstackClientCtl.session.loading">
-                <oui-spinner></oui-spinner>
+                <oui-loader size="m" class="d-inline-block"></oui-loader>
```

Comment:
Errr component name as per ui-kit documentation is <oui-spinner size="s|m|l"></oui-spinner> :/

Reply:
what happen if you put oui-spinner?